client/gutenberg/editor/style.scss

```diff
@@ -108,4 +109,22 @@ $gutenberg-theme-toggle: #11a0d2;
 			z-index: 100000; // Above WP toolbar.
 		}
 	}
+
+	// MISSING CORE STYLES (TODO: Remove them once Gutenberg include the Core styles)
+	input[type='radio'] {
```

Comment:
Some Calypso specific styles are probably applied here too, not sure if they are causing any issues:

https://github.com/Automattic/wp-calypso/blob/59bdfeeb97eda4266ad39410cb0a074d2c88dbc8/assets/stylesheets/shared/_forms.scss#L138

Reply:
Fortunately, they're very similar to the [Core styles](https://github.com/Automattic/wp-calypso/blob/59bdfeeb97eda4266ad39410cb0a074d2c88dbc8/assets/stylesheets/shared/_forms.scss#L138-L145) and the [`@wordpress/edit-post` styles](https://github.com/WordPress/gutenberg/blob/master/packages/edit-post/src/style.scss#L217-L235) was overriding most of the differences, so I needed to add only a couple of rules that were mismatching. 

Now that I'm reverting the `@wordpress/edit-post` styles import, I'd probably need to have these Calypso specific styles in consideration.